package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "notion-translations",
   "main": "build/index.js",
-  "version": "1.5.1",
+  "version": "1.6.0",
```

Comment:
I'd prefer do bump this in a separate commit 😄

Reply:
Noted! :)